main.go

```
@@ -429,6 +429,13 @@ func runHeartbeat() {
 	}
 }
 
+func verifyDownload(destPath string) error {
```

golangci-lint (1.62.2)

func `verifyDownload` is unused

(unused)